pkg/placement/controller.go

```diff
@@ -75,17 +75,20 @@ var excludedResourceNames = map[string]bool{
 type Controller struct {
 	ctx              context.Context
```

Comment:
context should not be part of the controller struct. it should be received in Start function as arg.